datadog/resource_datadog_integration_aws_lambda_arn.go

```diff
@@ -82,39 +80,40 @@ func resourceDatadogIntegrationAwsLambdaArnExists(d *schema.ResourceData, meta i
 
 func resourceDatadogIntegrationAwsLambdaArnCreate(d *schema.ResourceData, meta interface{}) error {
 	providerConf := meta.(*ProviderConfiguration)
-	client := providerConf.CommunityClient
+	datadogClientV1 := providerConf.DatadogClientV1
+	authV1 := providerConf.AuthV1
 
 	attachLambdaArnRequest := buildDatadogIntegrationAwsLambdaArnStruct(d)
-	err := client.AttachLambdaARNIntegrationAWS(attachLambdaArnRequest)
-
+	_, _, err := datadogClientV1.AWSLogsIntegrationApi.CreateAWSLambdaARN(authV1).Body(*attachLambdaArnRequest).Execute()
 	if err != nil {
 		return translateClientError(err, "error attaching Lambda ARN to AWS integration account")
 	}
 
-	d.SetId(fmt.Sprintf("%s %s", *attachLambdaArnRequest.AccountID, *attachLambdaArnRequest.LambdaARN))
+	d.SetId(fmt.Sprintf("%s %s", attachLambdaArnRequest.AccountId, attachLambdaArnRequest.LambdaArn))
```

Comment:
Should these use the `Get`ters?

Reply:
Addressed here https://github.com/terraform-providers/terraform-provider-datadog/pull/498